cropduster/static/cropduster/js/cropduster.js

```diff
@@ -372,6 +372,20 @@ window.CropDuster = {};
                     $thumb.html($thumb.html() + $.render.cropdusterImage(thumbData[size.name]));
                 }
             });
+            if ($thumb) {
```

Comment:
For jquery truthiness, you'll want `if ($thumb.length)` here (an empty jQuery object still evaluates as truthy). Out of curiosity: what problem is this fixing?

Reply:
Eh, it was an unnecessary guard class. Didn't solve a problem and is now no longer not solving a problem :smile: